Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs

```diff
@@ -3,41 +3,50 @@
 // FREE TO USE TO CONNECT THE WORLD
 // ---------------------------------------------------------------
 
+using System;
 using System.Threading.Tasks;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.ChangeTracking;
 using Taarafo.Core.Models.GroupPosts;
 
 namespace Taarafo.Core.Brokers.Storages
 {
-	public partial class StorageBroker
-	{
-		public DbSet<GroupPost> GroupPosts { get; set; }
+    public partial class StorageBroker
+    {
+        public DbSet<GroupPost> GroupPosts { get; set; }
 
-		public async ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost)
-		{
-			using var broker =
-				new StorageBroker(this.configuration);
+        public async ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost)
+        {
+            using var broker =
+                new StorageBroker(this.configuration);
 
-			EntityEntry<GroupPost> groupPostEntityEntry =
-				await broker.GroupPosts.AddAsync(groupPost);
+            EntityEntry<GroupPost> groupPostEntityEntry =
+                await broker.GroupPosts.AddAsync(groupPost);
 
-			await broker.SaveChangesAsync();
+            await broker.SaveChangesAsync();
 
-			return groupPostEntityEntry.Entity;
-		}
+            return groupPostEntityEntry.Entity;
+        }
```

Comment:
This is the old way of doing the broker implementation.  You can change this to:

```
public async ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost)
        await InsertAsync(groupPost);
```